addons/mail/static/src/components/discuss_sidebar_category/discuss_sidebar_category.xml

```diff
@@ -3,19 +3,20 @@
 <templates xml:space="preserve">
     <t t-name="mail.DiscussSidebarCategory" owl="1">
         <t t-if="category">
+            <t t-set="o_DiscussSidebarCategory_hoverItem" t-value="'btn p-0 text-left text-900 opacity-100-hover opacity-75'"/>
```

Comment:
`text-900` seems to be darker, maybe use `text-700` or `text-800` instead?

Reply:
Fixed. Same approach as here --> https://github.com/odoo/odoo/pull/89908#discussion_r863937990